### PR TITLE
fix preview video bug

### PR DESCRIPTION
**Pull request by pldespaigne**
fix the bug that prevent to preview video from the file explorer

### CODE REVIEW

File: libs/media/src/lib/file/explorer/explorer.model.ts [hunk 1]
```diff
@@ -89,6 +89,16 @@ function getFormListStorage(object: { id: string }, collection: CollectionHoldin
   return FormList.factory<StorageFile>(value, file => new StorageFileForm(file));
 }
 
+function getFormStorageVideo(object: { id: string }, collection: CollectionHoldingFile, label: FileLabel) {
```
Comment (RemcoSimonides) [inline libs/media/src/lib/file/explorer/explorer.model.ts:92]: It wasn't possible to give the object a proper type?

**Review by RemcoSimonides — APPROVED** [review]
LGTM

File: libs/media/src/lib/file/explorer/explorer.model.ts (hunk 1 above)
Comment (pldespaigne) [reply libs/media/src/lib/file/explorer/explorer.model.ts:92]: it's not a type problem, the function `getFormStorage` is calling `new StorageFileForm` which does not create a field for `jwPlayerId` that's why I created a new function that instantiate the good form :)

Comment (RemcoSimonides) [reply libs/media/src/lib/file/explorer/explorer.model.ts:92]: Ah ye I see, but I was talking about `object: { id: string }`. Isn't one of them a StorageFile and the other one a StorageVideo?

**Pull request merged**